### PR TITLE
Additional unofficial icons

### PR DESCRIPTION
**Pull request by MikeHughey1**

### CODE REVIEW

**Comment by FatBoyXPC** [comment]
Yay! Finally got this working! :joy: What do you think of these @keemy?

**Comment by keemy** [comment]
mostly good 234relay and 2345relay are not centered (the other relays seem they are). If you are happy with them then 👍 .

still not a fan of miniguild but I guess it's ok for now and if we can think of something better later we can change it

**Comment by MikeHughey1** [comment]
I didn't realize that 234relay and 2345relay were not centered.  Let me fix
them - it's not hard to do.  I'll get them this morning if I have time; if
not, I'll get them done tonight.

**Comment by MikeHughey1** [comment]
Easier than I thought - I think they're centered now.

**Comment by jfly** [comment]
I feel like the amount of whitespace on the kilominx is pretty high. Is that intentional? Compare:

image

to say 222:

image

I also am not in love with the match the scramble icon, did you decide you didn't like the old icon with the "solved cube -> scrambled cube"?

**Comment by FatBoyXPC** [comment]
> I also am not in love with the match the scramble icon, did you decide you didn't like the old icon with the "solved cube -> scrambled cube"?

TBH, I think @keemy made it clear he didn't like it.

**Comment by MikeHughey1** [comment]
This time I tried to use spacing on kilominx to match that of megaminx,
square-1, and pyraminx.  I thought I had it pretty consistent with those -
is that not true?

And yes, my new match-the-scramble icon is the way @keemy indicated he
thought would work well, and I actually think it's nicer than the old one,
now that I've used it.

**Comment by jfly** [comment]
I was referring to the amount of whitespace bordering the whole thing, not the amount of whitespace between the stickers.

**Comment by MikeHughey1** [comment]
Oh okay, now I understand.  I suppose I can fix it - how big do you want it
to be?

I notice the megaminx has more whitespace at the top than at the bottom.
My kilominx has more whitespace at the bottom than at the top.  So I guess
that's inconsistent.

It makes sense to me that the kilominx should be slightly smaller than the
megaminx - the 2x2x2 is slightly smaller than the 3x3x3.

So should I match the relative ratio that exists between 2x2x2 and 3x3x3
for kilominx vs. megaminx?  And what about the whitespace?  Should I
basically try to center the kilominx within the megaminx's footprint?  (In
other words, should I move it down a bit?)

On Fri, Oct 27, 2017 at 5:39 PM, Jeremy Fleischman <notifications@github.com

**Comment by jfly** [comment]
I like the idea of centering the kilominx within the megaminx's footprint!

**Comment by MikeHughey1** [comment]
Okay, I centered it.  I'm hoping it's good enough - it looks pretty good to me.  In actuality it's a little smaller in ratio compared to the megaminx than is the 2x2x2 vs. the 3x3x3, but it's pretty close to the same - only a little smaller.  And because I've just calculated the points of the kilominx to generate this (mostly by hand - silly perhaps, but it's what I did), it would be a fair amount of work to expand it a little bit to make it match the ratio for 2x2x2 vs. 3x3x3.  So is it good enough now?

**Comment by jfly** [comment]
Ok, this looks good to me! @keemy, @FatBoyXPC I'll merge this up on Friday unless one of you objects or merges it up beforehand.

**Comment by FatBoyXPC** [comment]
No objections here! :)

**Comment by keemy** [comment]
:shipit:

**Comment by MikeHughey1** [comment]
Thanks, guys!

On Fri, Nov 3, 2017 at 11:43 AM, Jeremy Fleischman <notifications@github.com
